### DISCUSSION TITLE
Remove the AI generated images?

### DISCUSSION THREAD

**Discussion by friadev** [HotCakeX/Harden-Windows-Security#203] [General]
Awesome guide but, I find the AI images off-putting. If you got rid of them it would look much nicer and I wouldn't have to deal with a surprise catgirl with a malformed eyeball.

**Comment by HotCakeX** [comment]
Hi,
Thank you for your compliment and feedbacks, based on them I've done the following:

1. Reduced the dimensions of the images on the readme.
2. Changed the images that had deformities with new ones that don't have those issues.
3. Replaced some old images with new ones that don't include a person.

I don't want to completely remove all images from the readme because they are part of the design and without them pure text is too boring, but I will pay more attention to the images I set so they won't have those problems anymore and will have higher standard/quality bar.

**Reply by friadev** [reply]
Very much appreciated!
